index.html

```diff
@@ -1,12 +1,476 @@
+                        </p>
+                    </div>
+                    <button class="white-button"><a class="button-text">Start the Conversation </a></button>
+                </div>
+
+            </div>
+        </section>
+
+
+        <section class="two-columns section-margin">
+            <div class="container">
+
+                <div class="content">
+                    <div class="content-title">
+                        <h2>We <a class="blue-text">Know</a> Talent Management</h2>
+                    </div>
+
+                    <div class="flex">
+                        <div class="column">
+                            <div class="column-img">
+                                <img src="img/illustration-happy-people.svg" />
+                            </div>
+                        </div>
+
+                        <div class="column">
+                            <div class="column-subtitle">
+
+                            </div>
+                            <div class="column-desc">
+                                <p>We get it. Engaging and retaining talent isn’t just hard in today’s competitive
+                                    marketplace
+                                    - it can feel downright impossible. When it comes to an engaged workforce, there are
+                                    no
+                                    one-size-fits-all solutions. It was only through our research that it all came back
+                                    to one
+                                    fundamental truth: human connection drives engagement. Connection to our peers, to
+                                    our
+                                    managers, and most importantly, to the work we do.</br>
+                                    </br>
+                                    Avenue is dedicated to connecting these dots. Our expert facilitators,
+                                    science-backed
+                                    leadership tools and proprietary platform gives employees the insight, courage, and
+                                    guidance
+                                    they need to maximize their impact within your organization.</p>
+                            </div>
+                            <div class="column-btn">
+                                <button class="button-learn-more"><a class="button-learn-more-text">Learn more</a></button>
+                            </div>
+                           
+                        </div>
+                    </div>
+
+                </div>
+
+        </section>
+        
+        <section class="three-columns section-margin">
+            <div class="container">
+                <div class="content">
+                    <div class="content-title">
+                        <h2><span class = "blue-text">Root Causes </span> of Career Apathy</h2>
+                    </div>
+                    <div class="flex">
```

Comment:
`flex` is a helper class. It's not saying a lot about block itself. To help other developers understand what's going on here - add more classes which explaining layout significant properties. 
For ex. 
```
three-columns
  content
    content-title
    columns
      column
      column
      column
```
and only then you can add helper classes like `flex` - when html classes already corresponding with the widget layout.